101._symmetric_tree.md

```diff
@@ -0,0 +1,82 @@
+## Symmetric Tree
+
+### Description
+
+```Python
+Given a binary tree, check whether it is a mirror of itself (ie, symmetric around its center).
+
+For example, this binary tree [1,2,2,3,4,4,3] is symmetric:
+
+    1
+   / \
+  2   2
+ / \ / \
+3  4 4  3
+
+But the following [1,2,2,null,3,null,3] is not:
+    1
+   / \
+  2   2
+   \   \
+   3    3
+Note:
+Bonus points if you could solve it both recursively and iteratively.
+```
+
+### Solutions
+
+#### Approach 1: Recursive
+
+* Time complexity : O(n)
+* Space complexity : O(n)
+
+```Python
+class Solution(object):
+    def isSymmetric(self, root):
```

Comment:
We usually name a method with `_` in python code style.
I think `is_symmetric` will be better.